docs/content/v1/usage/_index.md

```diff
@@ -3,16 +3,15 @@ title: Usage
 weight: 3
 ---
 
-A web-component can optionally have 3 parts: some HTML in a `<template>` tag, some JavaScript
-in a `<script>` tag and some CSS in a `<style>` tag.
+A Web Component can optionally have 3 parts:
 
-You can make a web-component for multiple reasons.
+* some HTML in a `<template>` tag,

```

Comment:
For me, it's clearer to use bullet points. Tell me if not :slightly_smiling_face: